pkg/types/library.go

```diff
@@ -1,5 +1,12 @@
 package types
 
+import "github.com/aquasecurity/trivy-db/pkg/db"
+
+type VersionInfo struct {
+	TrivyVersion           string      `json:",omitempty"`
+	VulnerabilityDBVersion db.Metadata `json:",omitempty"`
```

Comment:
Type in db.Metadata is just int. If we display it, should we convert it to string such as "light".

Reply:
```suggestion
	VulnerabilityDB db.Metadata `json:",omitempty"`
```
I feel `Version` is not necessary here.

Reply:
Fixed here https://github.com/aquasecurity/trivy/pull/435/commits/3b753943b594da309ab74dafff74c8aa13cab62b

Reply:
type names fixed here: https://github.com/aquasecurity/trivy/pull/435/commits/ea67f8c0e77508a6df4b22bde08b6d9a4faf9694